### PR TITLE
added BlendOS

### LINKED ISSUE #640: Add BlendOS
closed by #639

### PR DESCRIPTION
**Pull request by philclifford**
Closes #640 

https://github.com/quickemu-project/quickemu-icons/issues/12

### CODE REVIEW

**Comment by philclifford** [comment]
Todo: 
 - [ ] test installing
 - [ ] dynamically fetch releases (WIP)

File: quickget [hunk 1]
```diff
@@ -263,7 +263,8 @@ function editions_arcolinux() {
 }
 
 function releases_blendos() {
-    echo 23.01
+    curl -s https://api.github.com/repos/blend-os/blendOS/releases  |grep tag_name | grep -o -E '[[:digit:]]+\.[[:digit:]]+'
```
Comment (philclifford) [inline quickget:266]: @RudraSwat does this look reasonably sane for catching your future release tags so QuickEmu can pick them up dynamically ?

Comment (RudraSwat) [reply quickget:266]: @philclifford yep, lgtm +1 (I'll be sticking to this format for future release tags)

**Review by RudraSwat — APPROVED** [review]

**Comment by ben-alkov** [comment]
Sadly, the 23.04 release ISOs are at SourceForge[1], so installation attempts simply fail with a completely opaque error

```
quickget blendos 23.04
grep: warning: stray \ before "
grep: warning: stray \ before "
Exception caught
Exception: [download_helper.cc:451] errorCode=1 Unrecognized URI or unsupported protocol:


ERROR! Failed to download  with aria2c. Try running 'quickget' again.
```

[1] As seen [here](https://github.com/blend-os/blendOS/releases/tag/23.04)... e.g. https://sourceforge.net/projects/blendos/files/23.04-1/Plasma/blendOS-2023.04.22-x86_64-plasma.iso/download

**Comment by philclifford** [comment]
@ben-alkov Thanks - at least it auto-detected a new release. It was unaware of the release not being on github though, but there's a fix nearly ready.

**Pull request merged**